.github/workflows/reusable_dispatch_analytics.yml

```diff
@@ -3,10 +3,40 @@ name: Reusable dispatch analytics
 on:
   workflow_dispatch:
 
+  schedule:
```

Comment:
Perhaps split into a dedicated scheduled workflow so reusable remains a dispatch that can be called like other reusables?

Reply:
I've moved this to a normal workflow, and removed the reusable part.